checks/rabbitmq/check.go

```diff
@@ -105,10 +105,6 @@ func New(config Config) func(ctx context.Context) error {
 
 			// release the channel resources, and unblock the receive on done below
 			close(done)
-
-			// now drain any incidental remaining messages
-			for range messages {
-			}
```

Comment:
Please make this change in a separate PR, to keep the context only about SQL Server

Reply:
Sure!!